FreeRTOS/Source/portable/GCC/RISC-V/port.c

```diff
@@ -267,4 +269,3 @@ void vPortEndScheduler( void )
 
 
 
-

```

Comment:
Please keep, even if it's stupid.

Reply:
hmmm, that was adding after committing the GitHub suggestion, I'll delete it and squash into one commit